Modules/Sources/WordPressUI/Views/SectionHeader.swift

```diff
@@ -0,0 +1,16 @@
+import SwiftUI
+import DesignSystem
+
+public struct SectionHeader: View {
```

Comment:
Any reason not to use the default header style in `Section("Header text") {}`?

Reply:
It makes it a bit easier to read, which we want in a form, and it's more consistent with the WordPress design. I plan to keep using these for forms.